ape_etherscan/client.py

```diff
@@ -456,10 +459,11 @@ def _get_page_of_normal_transactions(
         }
         result = self._get(params=params)
 
-        if not isinstance(result.value, list):
-            raise UnhandledResultError(result, result.value)
+        value = result.value or []
```

Comment:
another fix: sometimes this is `None` and that is still 200 / ok.
for example, give it the ZERO_ADDRESS and you get this.